mkosi/__init__.py

```diff
@@ -4666,6 +4696,15 @@ def __init__(self, *kargs: Any, **kwargs: Any) -> None:
         kwargs["fromfile_prefix_chars"] = ArgumentParserMkosi.fromfile_prefix_chars
         super().__init__(*kargs, **kwargs)
 
+    @staticmethod
+    def _camel_to_arg(camel: str) -> str:
+        s1 = re.sub("(.)([A-Z][a-z]+)", r"\1-\2", camel)
+        return re.sub("([a-z0-9])([A-Z])", r"\1-\2", s1).lower()
+
+    @classmethod
+    def _ini_key_to_cli_arg(cls, key: str) -> str:
+        return cls.SPECIAL_MKOSI_DEFAULT_PARAMS.get(key, '') or ('--' + cls._camel_to_arg(key))
```

Comment:
```suggestion
        return cls.SPECIAL_MKOSI_DEFAULT_PARAMS.get(key) or ('--' + cls._camel_to_arg(key))
```
Setting the default of `get` to an empty string should be unnecessary, since `None` is falsey and will trigger the other branch of the `or`

Reply:
So it is. I was convinced that that'd raise an exception. Duh.

Reply:
`mypy` was unhappy with `.get(key, None)`, so I changed that to `.get(key, '')`. But `.get(key)` is obviously nicer, and `mypy` likes it too.